Clinical-Genomics/BALSAMIC
# Disable auto trimming of reads

This PR disables auto trim from fastq files

To test this PR:
- All tests should pass

Expected output:
- Changes to config file with four extra flags in analysis config and QC section:

```
 "quality_trim": "False",
 "adapter_trim": "False",
"umi_trim": "False",
"umi_trim_length": "5"
```
- New cli option to enable and disable above flags.
